### DISCUSSION TITLE
How to use the benchmark.py script?

### DISCUSSION THREAD

**Discussion by SnoozingSimian** [triton-inference-server/fastertransformer_backend#65] [Q&A]
I couldn't find any documentation regarding how to use the `benchmark.py` script for testing my GPT-triton server?

Is it at all possible? What do I need to? What is the expected output?

**Comment by byshiue** [comment]
It is an old tool and not maintained now. You can follow the document to run the test, and use random inputs with different shapes to run benchmark.
